api/grpcserver_test.go

```diff
@@ -940,7 +941,11 @@ func TestGrpcServer_GetActions(t *testing.T) {
 		}
 
 		svrDisableIndex := svr
-		svrDisableIndex.core.hasActionIndex = false
+		coreService, ok := svrDisableIndex.core.(*coreService)
+		if !ok {
+			require.Error(err)
+		}
+		coreService.hasActionIndex = false
```

Comment:
Add a TODO to create a core service with action index disabled, instead of updating the `hasActionIndex` here

Reply:
yes